src/database/migrations/20191031090043-add-isAvailable-column-to-accommodations-table.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
```

Comment:
Thank you @urbanishimwe for the job. However, we already have this migration file. try to rebase